go.mod

```diff
@@ -343,6 +345,8 @@ require (
 )
 
 replace (
+	github.com/celestiaorg/celestia-app/v3 v3.0.0 => ../celestia-app
+	github.com/celestiaorg/go-square/v2 v2.0.0 => ../go-square
```

Comment:
We already have v2.0.0 ;) https://github.com/celestiaorg/go-square/releases/tag/v2.0.0

Reply:
I know. But it does not have my local changed ;)